### PR TITLE
DDF-3327 Fix description overflow in Admin UI Application boxes

### PR DESCRIPTION
**Pull request by kcrowe01**
#### What does this PR do?
Fixes a problem where the description of an application would overflow the box it was held in depending on the size of the user's window.
#### Who is reviewing it? 
@mcalcote @vinamartin 
#### Choose 2 committers to review/merge the PR. 
@clockard 
@rzwiefel 
#### How should this be tested? (List steps with links to updated documentation)
Install DDF and navigate to the admin console. Verify that resizing your browser does not cause the description text for applications to overflow the application boxes. This should be tested on IE10, chrome and firefox.
#### What are the relevant tickets?
[DDF-3327](https://codice.atlassian.net/browse/DDF-3327)
#### Screenshots (if appropriate)
screen shot 2017-09-28 at 10 23 17 am
screen shot 2017-09-29 at 12 33 33 pm

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/admin/ui/src/main/webapp/less/application/app-card.less [hunk 1]
```diff
@@ -25,6 +25,8 @@
   .panel-body {
     padding: 0px @app-card-overlay-width 0px 0px; // reset bootstrap
     height: 100%;
+    display: flex;
```
Comment (clockard) [inline platform/admin/ui/src/main/webapp/less/application/app-card.less]: Is flex supported on all of they browsers we support. For some reason I thought it wasn't. @andrewkfiedler would know.

**Comment by clockard** [comment]
Since this is a UI change can you add a screenshot? Maybe even a before and after?

File: platform/admin/ui/src/main/webapp/less/application/app-card.less (hunk 1 above)
Comment (kcrowe01) [reply platform/admin/ui/src/main/webapp/less/application/app-card.less]: @clockard Is there somewhere I can find a list of supported browsers?

Comment (clockard) [reply platform/admin/ui/src/main/webapp/less/application/app-card.less]: I don't know that there is an exhaustive list but the docs say the minimum browser requirements are IE 10. I think for this particular change, IE would be the only browser of concern.

**Comment by kcrowe01** [comment]
build now

**Comment by vinamartin** [comment]
Is the IE10 issue still a concern? Can someone manually verify this?

**Comment by kcrowe01** [comment]
I verified against IE10 but it should still be tested as part of the hero process.

**Comment by rzwiefel** [comment]
We should get @andrewkfiedler to look at this 😄

**Comment by rzwiefel** [comment]
Pinging @andrewkfiedler again for this.

**Pull request merged**